gcc/rust/hir/rust-hir-dump.cc

```diff
@@ -236,25 +269,43 @@ void
 Dump::visit (BlockExpr &block_expr)
 {
   stream << "BlockExpr: [";
+
   indentation.increment ();
-  stream << std::endl;
   // TODO: inner attributes
+  if (!block_expr.inner_attrs.empty ())
+    {
+      stream << "\n";
+      stream << indentation << "inner_attrs: [";
+      for (auto &attr : block_expr.inner_attrs)
+	{
+	  stream << "\n";
+	  stream << indentation;
+	  visit (attr);
+	}
+    }
 
   // statements
+  // impl null pointer check
+  stream << "\n";
```

Comment:
I think this should be before the brace on line 285 in case there are no attributes.

Reply:
In my opinion. If there's no attrs. Stmts will dump in a new line instead of the line with "BlockExpr: [".

Reply:
You are right, it was just a little hard to see. Moving line 277 before the if (after 247) and line 289 after 284 would do the same (right?) and would be easier to follow.

Reply:
I add "\n" behind {} and []. Now seems more clear.

Reply:
Thanks! Much better.